rhombus-lib/rhombus/private/amalgam/exn-object.rkt

```diff
@@ -74,27 +74,27 @@
               Variable))
 
 (define-exn Arity exn:fail:contract:arity
-  #:parent Contract exn:fail:contract
+  #:parent Annot exn:fail:contract
   #:fields ()
   #:children ())
 
 (define-exn DivideByZero exn:fail:contract:divide-by-zero
-  #:parent Contract exn:fail:contract
+  #:parent Annot exn:fail:contract
   #:fields ()
   #:children ())
 
 (define-exn NonFixnumResult exn:fail:contract:non-fixnum-result
-  #:parent Contract exn:fail:contract
+  #:parent Annot exn:fail:contract
   #:fields ()
   #:children ())
 
 (define-exn Continuation exn:fail:contract:continuation
-  #:parent Contract exn:fail:contract
+  #:parent Annot exn:fail:contract
   #:fields ()
   #:children ())
 
 (define-exn Variable exn:fail:contract:variable
-  #:parent Contract exn:fail:contract
+  #:parent Annot exn:fail:contract
```

Comment:
It seems to me that these exceptions make less sense as “sub-exceptions” of `Exn.Annot`.

Reply:
I see what you mean, and perhaps `Contract` is the right word in the sense that it's the run-time component of an annotation. It seemed awkward to have `~exn: Exn.Fail.Contract` but `annot` in all other parts, though. Maybe `Exn.Fail.Annot` should be a constructor that creates an `Exn.Fail.Contract` — that is, an alias for construction purposes, but not for  purposes like `is_a` (to avoid suggesting that an `Exn.Fail.Contract.Variable` instance, say, is a `Exn.Fail.Annot`)?